test/DynamoCoreTests/DSEvaluationViewModelTest.cs

```diff
@@ -1113,6 +1113,19 @@ public void TestDefaultValueAttribute()
         }
 
         [Test]
+        public void TestDefaulArgumentAttributeNegative()
+        {
+            // This is to test FFITarget.TestData.MultiplyBy3NonParsableDefaultArgument() whose
+            // DefaultArgumentAttribute is invalid. In this case, we should make sure that
+            // no default argument is used, even null. So this function should be compiled to
+            // a function object and Apply() should work on it. 
+            DynamoUtilities.DynamoPathManager.Instance.AddPreloadLibrary("FFITarget.dll");
```

Comment:
I'm guessing your test case would pass without loading this library (something odd that I noticed). Anyway, `DynamoPathManager` is dead, if you need the library to be preloaded, please go through `DynamoViewModelUnitTest.pathResolver` in the base class, that resolver is supposed to return `FFITarget.dll` (I wasn't sure how to do that, because `pathResolver` is created way before this test case in `Setup` method).